fl4health/parameter_exchange/layer_exchanger.py

```diff
@@ -92,88 +91,76 @@ def pull_parameters(self, parameters: NDArrays, model: nn.Module, config: Option
         model.load_state_dict(current_state, strict=True)
 
 
-class NormDriftParameterExchanger(ParameterExchangerWithPacking[List[str]]):
+class DynamicLayerExchanger(PartialParameterExchanger[List[str]]):
```

Comment:
Not a huge deal, but curious why you changed the name of this class. It still seems be accurate to call it the NormDriftParameterExchanger. Is DynamicLayerExchanger potentially too general of a name (ie I would expect you can have a DynamicLayerExchanger that does not use a norm based selection criteria. Maybe I am wrong so let me know :)

Reply:
So I changed the name for two reasons. The first is that I want to emphasize that this exchanger only exchange entire layers, and the second is that now the exchanger can take in customized layer selection criterion function, which may not select layers based on their drift in l2 norms. I agree it's a little unclear what "DynamicLayerExchanger" refers to, but I couldn't think of a better name 😂  If you have other suggestions please let me know!